pkg/cloudprovider/amifamily/ami.go

```diff
@@ -155,7 +155,7 @@ func (p *AMIProvider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[stri
 	p.ec2Cache.SetDefault(fmt.Sprint(hash), output.Images)
 	amiIDs := lo.Map(output.Images, func(ami *ec2.Image, _ int) string { return *ami.ImageId })
 	if p.cm.HasChanged("amiIDs", amiIDs) {
-		logging.FromContext(ctx).Debugf("Discovered images: %s", amiIDs)
+		logging.FromContext(ctx).With("amiIDs", amiIDs).Debugf("Discovered images")
```

Comment:
Can you do a pass to make sure we're using consistent casing? 
```suggestion
		logging.FromContext(ctx).With("ami-ids", amiIDs).Debugf("Discovered images")
```